dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/conf/ActivationLayerConfigurationBuilder.java

```diff
@@ -26,6 +26,7 @@
  * Configuration builder for activation layer.
  *
  * The configuration that this builder generates is used to create an activation layer instance.
+ * The generate configuration need to bind the implementation for matrix factory, in order to inject layer instance.
```

Comment:
Sorry for confusing you. They need to bind `LayerInputShape`, not `matrixFactory`. I'll fix it.